Formula/cockroach.rb

```diff
@@ -38,9 +38,9 @@ def caveats; <<-EOS.undent
   plist_options :manual => "cockroach start --insecure"
 
   def plist; <<-EOS.undent
-    <?xml version="1.0" encoding="UTF-8"?>
-    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-    <plist version="1.0">
+    <?xml version="1.0.1" encoding="UTF-8"?>
```

Comment:
Ha! Looks like a bug in `brew bump-formula-pr`. Welp. Looks like it'll only happen this once.